### PR TITLE
Registration form polishing

### PR DESCRIPTION
**Pull request by lxkuz**
### Открытая регистрация:

**iphone 5/se**
Screenshot 2021-05-04 at 10 24 33

**iphone 7/8**
Screenshot 2021-05-04 at 10 24 42

### Регистрация в фонд

**iphone 5/se**

Screenshot 2021-05-04 at 10 24 57

**iphone 7/8**

Screenshot 2021-05-04 at 10 24 49

### Ошибки валидации
Screenshot 2021-05-04 at 10 25 32

### CODE REVIEW

File: app/views/devise/shared/_registration_links.html.erb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<%- if devise_mapping.confirmable? && controller_name != 'confirmations' %>
+  <%= link_to "Не пришло подтверждение?", new_confirmation_path(resource_name), style: 'font-size: 10px' %><br />
+<% end %>
+
+<%- if devise_mapping.lockable? && resource_class.unlock_strategy_enabled?(:email) && controller_name != 'unlocks' %>
+  <%= link_to "Не пришли инструкции по разблокировке?", new_unlock_path(resource_name), style: 'font-size: 10px'  %><br />
+<% end %>
```
Comment (memumen) [inline app/views/devise/shared/_registration_links.html.erb:7]: Please add new line

File: app/views/shared/_registration_steps.html.erb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+<% step_number = steps[step] %>
+<div class="registration-steps mb-1" style="font-size: 9px">
+  <%= steps.map do |step, number|
+    css = ""
+    css += "color: grey;" if number <  step_number
+    css += "font-weight: bold;" if number == step_number
+    label = t("registration_steps.#{step}")
+    "<span style='#{css}'>#{label}</span>"
+  end.join(" -> ").html_safe %>
+</div>
```
Comment (memumen) [inline app/views/shared/_registration_steps.html.erb:12]: same here

File: app/views/shared/_registration_steps.html.erb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<% steps = { form: 1, confirm_email: 2, waiting_for_moderator: 3, good_to_go: 4 } %>
+<% step ||= :form %>
+<% step_number = steps[step] %>
+<div class="registration-steps mb-1" style="font-size: 9px">
+  <%= steps.map do |step, number|
+    css = ""
```
Comment (memumen) [inline app/views/shared/_registration_steps.html.erb:6]: Not a big deal, but why didn't you use the helper?

File: config/locales/ru.yml [hunk 4]
```diff
@@ -485,3 +485,8 @@ ru:
   notifications:
       help_request:
         create: Появились новые просьбы
+  registration_steps:
+    form: Отправка формы регистрации
+    confirm_email: Подтверждение почты
+    waiting_for_moderator: Проверка модератором фонда
+    good_to_go: Работа в приложении Helphub
```
Comment (memumen) [inline config/locales/ru.yml:492]: Please add new line

**Review by memumen — CHANGES_REQUESTED** [review]
Great job. Just a few comments.

File: app/views/shared/_registration_steps.html.erb (hunk 3 above)
Comment (lxkuz) [reply app/views/shared/_registration_steps.html.erb:6]: yeah I agree but this module will 100% be re-written when we get UI design, so let's keep as is

**Review by memumen — APPROVED** [review]

**Pull request merged**